src/events_microformats.js

```diff
@@ -131,10 +131,14 @@ function detectHCalendarEvents() {
     }
 
     event = utils.processEvent(event);
-    events.push(event);
 
-    // Insert a button inline near the title of the page.
-    $(vevent).find('.summary').prepend(getInlineIconSmall_(event));
+    var now = moment().valueOf();
```

Comment:
Please inline this; the fewer variables that are in scope at any given time, the better.